packages/relay/src/RelayServer.ts

```diff
@@ -164,7 +165,7 @@ export class RelayServer extends EventEmitter {
     const expiredInBlocks = parseInt(req.relayRequest.request.validUntil) - currentBlockNumber
     if (expiredInBlocks < this.config.requestMinValidBlocks) {
       throw new Error(
-          `Request expired (or too close): expired in ${expiredInBlocks} blocks, we expect it to be valid for ${this.config.requestMinValidBlocks}`)
+        `Request expired (or too close): expired in ${expiredInBlocks} blocks, we expect it to be valid for ${this.config.requestMinValidBlocks}`)
```

Comment:
If you get to it, avoid whitespace line changes. Nothing but merge conflicts come form these :-)

Reply:
Yes, that's my IDEA formatting...